### ISSUE TITLE
Display a list of all tickets in the AMT backlog

### ISSUE THREAD

**Issue by ChrisKellaleaMaynard** [aehrc/snomio#145]
**As a:** Terminology Author,  
**I want to:**  view a filterable and sortable list of tickets from the AMT backlog
**So I can:**  manage the AMT backlog of tickets.

---

### Related issues:
* #140 
* #123 
* #142 

### Acceptance Criteria

1. [If I do A.]
1. [B should happen.]

[
Also, here are a few points that need to be addressed:

1. Constraint 1;
1. Constraint 2;
1. Constraint 3.
]

### Resources:

* Mockups: [Here goes a URL to or the name of the mockup(s) in inVision];
* Testing URL: [Here goes a URL to the testing branch or IP];
* Staging URL: [Here goes a URL to the feature on staging];

### Notes

[Some complementary notes if necessary:]

* > Here goes a quote from an email
* Here goes whatever useful information can exist…

**Comment by kylynnloi** [comment]
@clinnygee  looking gewd - can we please make the Schedule and black triangle columns in the table read-only (they will be editable in the ticket details page though) and change the name of the column from black label to black triangle?

**Comment by clinnygee** [comment]
@kylynnloi Yes, will talk tomorrow at stand up - these two columns are broken from the db change, so he was going to fix them. Might be the best idea to push that stuff and break them - then I fix them from there. But we will talk about it tomorrow.

**Comment by HymaKanchiAU** [comment]
Verify that the following columns are displayed in the Backlog Page
Note the AC needs to be updated.
Priority - Pass
Title - Pass
Schedule - Pass
Iteration - Pass
Status - Pass
Labels - Pass
Assignee - Pass
Task number associated - Unable to verify as pending development.
Created Date (System generated at the moment and will be based on the DB created date) - Pass.

Ensure that Deleted status is not available in the status dropdown values. = Pass
Only the following options are available. = Pass
To Do
Closed
Review
Second Review/Author
Validation/First Review
Reopen
Resolved
Awaiting Confirmation

Ensure that the Filter has all the column names listed in bullet point 1 in the dropdown. = Missing Task number from the list 
Filter on each of the column names with the applicable filter and validate the results - should search on all the entries and not just on the entries that are displayed on screen. = Fail 
                           - Filter is performed only on the entries displayed on that page. 
                           - Unable to filter on Schedule as the values are not available.
                           - Unable to filter on Priority as this is not yet implemented.
                           - Unable to perform specific filter options for Labels (is any of, starts with, ends with etc)
                           - Unable to perform filter on Assignee = unassigned 
                           - Unable to perform filter on Created (is any of, starts with, ends with etc)
Search on each column name and ensure that the results are displayed. Unable to search on the following column names = Fail

                         - Schedule
                         - Iteration
                         - Labels
                         - Created
                         - Status
                         - Created

Priority there is no data available to perform the search
Assignee and Title have bugs -https://github.com/aehrc/snomio/issues/282, https://github.com/aehrc/snomio/issues/308

Search on a combination of column names and ensure that the results are displayed. - dependant on the above test results.

**Comment by ChrisKellaleaMaynard** [comment]
Removed deletion test case from this ticket.
